content/intro-to-storybook/vue/zh-CN/get-started.md

```diff
@@ -0,0 +1,73 @@
+---
+title: 'Storybook Vue 教程'
+tocTitle: '开始吧'
+description: '在你的开发环境下配置Storybook'
+commit: '9e3165c'
+---
+
+Storybook 是在开发模式下与您的应用程序一同运行的。它可以帮助您构建 UI 组件，并且将其与您应用程序中的业务逻辑和上下文分离开来。这份教程适用于 Vue；其他的则适用于[React](/react/en/get-started)，[React Native](/react-native/en/get-started/)，[Angular](/angular/en/get-started)，[Svelte](/svelte/en/get-started)和[Ember](/ember/en/get-started)。
+
+![Storybook and your app](/intro-to-storybook/storybook-relationship.jpg)
+
+## 设置 Vue Storybook
+
+我们将会通过下述的几个步骤来设置我们的环境。首先，我们使用[degit](https://github.com/Rich-Harris/degit)来设置我们的构建系统。您可以通过使用此包下载“模板”(一个包含了某些默认设置的部分构建好的应用程序)，来帮助您快速跟踪您的开发流程。
+
+让我们运行下述的命令：
+
+```bash
+# 克隆模板
+npx degit chromaui/intro-storybook-vue-template taskbox
+
+cd taskbox
+
+# 安装依赖
+yarn
```

Comment:
@zydmayday could you revert the comments in the snippets for this page to their native form as well?